ab_decrypt.py

```diff
@@ -20,6 +21,16 @@
 from struct import pack
 import ctypes
 
+def inputtty(prompt=""):
+  if platform.system() == "Windows":
+    return input(prompt)
```

Comment:
I haven't tested this, but I presume it should work.

Reply:
Thank you for all improvements on this code!

Reply:
I wish crediting you in readme. Should I use "crass@github" or other name ?

Reply:
You can use my real name since its already in the commit log, Glenn Wasbhurn (@crass). And thanks! Even though you did most of the hard work in putting the bits of the puzzle together.